tests/modules/committee_sites/test_committee_sites.py

```diff
@@ -0,0 +1,41 @@
+"""
+Tests donut/modules/committee_sites/
+"""
+from datetime import date
+import flask
+import pytest
+from donut.testing.fixtures import client
+from donut import app
+import donut.modules.core.helpers as core_helpers
+from donut.modules.committee_sites import helpers
+from donut.modules.committee_sites import routes
```

Comment:
You can combine these two imports: `from donut.modules.committee_sites import helpers, routes`